lib/ae_mdw/db/format.ex

```diff
@@ -365,6 +387,12 @@ defmodule AeMdw.Db.Format do
   defp custom_encode(_state, _tx_type, tx, _tx_rec, _signed_tx, _txi, _block_hash),
     do: tx
 
+  defp maybe_base64_pointer_key(key)
+       when key in ["account_pubkey", "oracle_pubkey", "contract_pubkey", "channel_pubkey"],
```

Comment:
Contrary to intuition, `channel_pubkey` should be just `channel`, as per [this implementation](https://github.com/aeternity/aeternity/blob/a5b4cfbf3f2afb30ae6cb06f0b21858a06652bb2/apps/aens/src/aens.erl#L38)